packages/client/e2e/pages/CreditRfqTilesPageObject.ts

```diff
@@ -0,0 +1,43 @@
+import { Page } from "@playwright/test"
+
+export class CreditRfqTilesPageObject {
+  constructor(readonly page: Page) {}
+
+  // TODO - these tiles filter functions will not work in squashed resonsive mode
+  selectLiveRfqsFilter() {
+    return this.page.getByText(/Live/).first().click()
```

Comment:
and let's fix this inconsistency too, while we are there - `getByText` or `getByTestId` .. not both